libc/test/src/search/lfind_test.cpp

```diff
@@ -9,10 +9,14 @@
 #include "src/search/lfind.h"
 #include "test/UnitTest/Test.h"
 
+namespace {
+
 int compar(const void *a, const void *b) {
   return *reinterpret_cast<const int *>(a) != *reinterpret_cast<const int *>(b);
 }
 
+}
+
```

Comment:
```suggestion
static int compar(const void *a, const void *b) {
  return *reinterpret_cast<const int *>(a) != *reinterpret_cast<const int *>(b);
}
```
Honestly, this is more in-line with LLVM's coding style, but I don't know how much `libc` differs.

Reply:
yeah, i'd have just used static myself but other llvm-libc tests seemed to have an anonymous namespace...